features/hook.feature

```diff
@@ -71,3 +71,34 @@ Feature: Tests `WP_CLI::add_hook()`
       `add_hook()` to the `before_invoke` is working.
       """
     And the return code should be 0
+
+  Scenario: Add callback to the `before_run_command` with args
```

Comment:
I'm definitely open to a better testing approach here to make sure the hook args are flowing through to the callback properly. 😆

Reply:
Testing looks good to me.